osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/monitoring/AbstractGetPowerQualityProfileHandler.java

```diff
@@ -184,7 +185,7 @@ private Map<CosemObject, List<CosemObject>> getProfilesToRead(
 
   private void addToMapIfNeeded(
       final DlmsObjectType profileType,
-      final String privateOrPublic,
+      final String pqProfile,
```

Comment:
To make this code more readable, the name `privateOrPublic` was deliberately choosen. The word `profile` is used in several different meanings, making it confusing to use.